grc/satellites_fixedlen_to_pdu.block.yml

```diff
@@ -35,7 +34,8 @@ outputs:
 templates:
     imports: |-
         import satellites
-    make: satellites.fixedlen_to_pdu(${type.size}, ${syncword_tag}, ${packet_len}, ${pack if type == 'byte' else False})
+        import satellites.grtypes
+    make: satellites.fixedlen_to_pdu(gr.types.vector_type(${type.vector_type}), ${syncword_tag}, ${packet_len}, ${pack if type == 'byte' else False})
```

Comment:
Is the explicit conversion to `gr.types.vector_type` necessary here? Comparing with Tagged Stream to PDU, I see that it [uses the types from `gr.types` directly](https://github.com/gnuradio/gnuradio/blob/main/gr-pdu/grc/pdu_tagged_stream_to_pdu.block.yml#L11) (and its constructor [also has a `gr::types::vector_type` parameter](https://github.com/gnuradio/gnuradio/blob/main/gr-pdu/include/gnuradio/pdu/pdu_to_tagged_stream.h)).

I suspect that this would also work without the explicit conversion to `gr.types.vector_type` and reduces the chances of breaking things under GR 3.9 or 3.8.

Reply:
I changed the PR accordingly. I can confirm it still works for GnuRadio 3.10, so the GnuRadio pybinds seem to have some explicit conversion from the types enum to vector_type.

I don't have a working GnuRadio 3.8 or 3.9 ready to test it over there, sorry. The whole point of me using gr-satellites over my old outdated custom OOT was to start on a blank and modern canvas.